File: app/src/main/java/io/twere/plainpure/data/api/dribbble/DribbbleProjectsService.java
package io.twere.plainpure.data.api.dribbble;

import io.twere.plainpure.mvp.model.dribbble.Project;
import io.twere.plainpure.mvp.model.dribbble.Shot;
import java.util.List;
import retrofit2.http.GET;
import retrofit2.http.Path;
import rx.Observable;

public interface DribbbleProjectsService {

  @GET("projects/{id}") Observable<Project> getProject(@Path("id") long projectId);

  @GET("projects/{id}/shots") Observable<List<Shot>> getShotsForProject(@Path("id") long projectId);
}

File: app/src/main/kotlin/io/twere/plainpure/data/api/DribbbleProjectsService.kt
package io.twere.plainpure.data.api

import io.twere.plainpure.data.local.project.Project
import io.twere.plainpure.data.local.shot.Shot
import retrofit2.http.GET
import retrofit2.http.Path
import rx.Observable

interface DribbbleProjectsService {

  @GET("projects/{id}")
  fun getProject(
      @Path("id") projectId: Long): Observable<Project>

  @GET("projects/{id}/shots")
  fun getShotsForProject(
      @Path("id") projectId: Long): Observable<List<Shot>>

}